source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -247,6 +253,21 @@ The following example performs these actions:
    `Dispose() <https://learn.microsoft.com/en-us/dotnet/api/system.idisposable.dispose?view=net-7.0>`__
    method once the cursor is no longer in use.
 
+.. _csharp-retrieve-instruqt-lab:
+
+Complete an Interactive Lesson
+------------------------------
+
+This lab helps you understand how to perform read operations in MongoDB
+by using the ``Find()`` method. You can complete this lab directly in
```

Comment:
Suggestion:

I think it could be helpful to identify the driver for similar reasons as the prior comment:
"by calling the Find() method in the {+driver-short+}"